src/tools/reverseEvent/reverse-event.ts

```diff
@@ -0,0 +1,73 @@
+import fs from "fs-extra";
+
+type logType = {
+  "_id": string,
+  "rid": string,
+  "command": number,
+  "args": string[]
+}
+
+async function reverseEvent() {
+  try {
+    const logs = await fs.readJson(__dirname + "/eventlog.json");
+
+    let args: {[key: string]: string}[] = [];
+
+    logs.forEach((log: logType) => {
+      let arg: {[key: string]: string} = {
```

Comment:
I think we do not need define type here.
Based on the monitor/src/substrate/handler/eventStorage.ts, 
command will be "CommandOp" type, and others are string.

Reply:
Yes, we do not need logType.